src/EnergyPlus/MixedAir.cc

```diff
@@ -4517,6 +4529,10 @@ void VentilationMechanicalProps::CalcMechVentController(
                 if (this->SystemOAMethod == SOAM_ProportionalControlSchOcc || this->SystemOAMethod == SOAM_ProportionalControlDesOcc ||
                     this->SystemOAMethod == SOAM_ProportionalControlDesOARate) {
                     SysOA = SysOA / SysEv;
+                } else if (this->SystemOAMethod == SOAM_VRP && this->SysDesOA > 0.0) {
+                    // Limit system OA to design OA minimum flow rate, as per ASHRAE Guideline 36-2018 Section 5.16.3.1
+                    // If no system sizing run is done (i.e. no Sizing:System) the design outdoor air flow rate is not known
+                    SysOA = min(SysOAuc / SysEv, this->SysDesOA);
```

Comment:
Ventilation optimization system outdoor air flow cap. We're only capping it when a system sizing run has been performed (i.e. design outdoor air flow rate is known through calculation or through user input).

Here's an example of results before and after the implementation of the cap. In this example, one of the air terminal has its minimum air flow rate scheduled. Depending on the hourly schedule value, the system ventilation efficiency can be lower than the design value which triggers an increase in system outdoor air which is not wanted.

![image](https://user-images.githubusercontent.com/5149279/127891168-effb391c-97a3-4746-ae06-6cf4f2024ce7.png)

Reply:
Actually I think we may need to be careful about this cap, for probably two reasons: first, the design OA minimum flow rate typically means that the system should not go below that value, but not a ceiling to cap the flow rate; secondly, the "normal" concern seems to be "worrying that the space does not get enough OA", rather than "getting too much OA". 

So here it might need to be carefully considered. In some sense, I even think using the "max" of these two might be reasonable, rather than using the "min" of the two values---so there might seems legitimate to keep those "fingers" in the graph above. Any input on this? @mjwitte @Myoldmopar @EnergyArchmage

Reply:
I'm not following the cap logic. The 90.1 spec talks about allowing the OA flow to be reduced *below* the design flow rate when `SysEv` is higher than design. That's already happening with the current code:
`SysOA = SysOAuc / SysEv;`.
And the current code will also increase the OA above the design flow rate when `SysEv` is degraded. Is there somewhere in Std 62.1 that says the OA flow rate should never go above the VRP design flow? That's what this cap is doing.